Modules/Chatroom/classes/class.ilChatroom.php

```diff
@@ -143,7 +143,8 @@ protected static function checkPermissions($usrId, $refId, array $permissions)
 			}
 			else
 			{
-				$hasAccess = $DIC->access()->checkAccessOfUser($usrId, $permission, '', $refId);
+				$hasAccess = $DIC->rbac()->system()->checkAccess($permission, $refId);
```

Comment:
I'm not sure about this. The `$usrId` is passed as an argument, why changing the method call here?

The comment should be removed.